src/diffpy/cmi/cli.py

```diff
@@ -72,6 +72,43 @@ def map_pack_to_examples() -> dict[str, List[str]]:
     return examples_by_pack
 
 
+def copy_examples(
+    examples: List[str],
+    target_dir: Optional[Path] = None,
+) -> List[Path]:
+    """Copy one or more examples to a target directory.
+
+    Parameters
+    ----------
+    examples : list of str
+        Example name(s): ['example1'], ['pack1']
+    target_dir : Path, optional
+        Target directory where examples should be copied.
+        Defaults to current working directory if not specified.
+
+    Returns
+    -------
+    list of Path
+        List of destination paths created.
+
+    Raises
+    ------
+    ValueError
```

Comment:
I don't think we want to raise this.  We just copy all versions that we find.  This will be a case we test.  

We often don't do this Raises section, the interesting raises are caught in the tests.  Maybe we should but our current style is not to.  It can be tmi